typescript/packages/wallets/viem/src/ViemEVMWalletClient.ts

```diff
@@ -42,6 +44,35 @@ export class ViemEVMWalletClient extends EVMWalletClient {
         };
     }
 
+    getCoreTools(): ToolBase[] {
```

Comment:
This should not be needed. The `WalletClientBase` already has this method :)

Reply:
Now removed - thanks!